app/controllers/events_controller.rb

```diff
@@ -0,0 +1,25 @@
+class EventsController < ApplicationController
+  def new; end
+
+  def index
+    @events = Event.all
```

Comment:
Create two extra variables for past and upcoming events:

```
past = Event.past
upcoming = Event.upcoming
```

You could refer to them in your views like this.

[`REQUIRED`]

References:
https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/past_upcoming_events_requirement.png

Reply:
we create this 2 extra variables for past and upcoming events